modules/nf-core/last/train/main.nf

```diff
@@ -14,6 +14,7 @@ process LAST_TRAIN {
     output:
     tuple val(meta), path("*.train"), emit: param_file
     path "versions.yml"           , emit: versions
```

Comment:
Please make the `versions` channel the last of the output.

Reply:
alright